DataManagementSystem/Client/FTSRequest.py

```diff
@@ -796,7 +793,7 @@ def __resolveFTSServer( self ):
 
     targetSite = ''
     for targetSite in targetSites:
-      targetFTS = getFTSServersForSites( [targetSite] )
+      targetFTS = getFTS2ServersForSites( [targetSite] )
```

Comment:
Where do this need to be checked exactly? I don't know much about this code, I let you change it.